rust/examples/dynamic_types.rs

```diff
@@ -40,7 +41,7 @@ fn main() {
     ]);
 
     // create some data
-    let id = PrimitiveArray::from(vec![1, 2, 3, 4, 5]);
+    let id = numeric_from_nonnull!(Int32Array, 1, 2, 3, 4, 5);
```

Comment:
IMO `Int32Array::from(vec![1, 2, 3, 4, 5])` is nicer, and more straightforward than a macro.
In the future I can imagine an `array!` macro instead.

Reply:
Thanks. I prefer the former too. The old impl no longer works though because `From<T::T>` and `From<Option<T::T>>` are conflicting implementations, perhaps because the compiler thought that `T::T` and `Option<T::T>` could overlap. I'm yet to figure out a way for this..